services/routes-google.js

```diff
@@ -3,55 +3,93 @@ const Toll = require('../models/tolls');
 const costTolls = require('./cost-tolls');
 const Vehicle = require('../models/vehicles');
 const findTollInSection = require('./searchFunctions').findTollInSection;
-const cleanFunctions = require('./cleanFunctions');
-const cleanPathFunction = cleanFunctions.cleanPathFunction;
+const validateArea = require('./area').limits;
+const redisClient = require('../app').redis;
 const missingTolls = require('./missingTollsFunction').missingTollsArray;
+// const cleanFunctions = require('./cleanFunctions');
+// const cleanPathFunction = cleanFunctions.cleanPathFunction;
+// const cleanTollsFunction = cleanFunctions.cleanTollsFunction;
+// const redisClient = require('../init_redis').redisClient;
+// const checkForMissingTolls = require('./searchFunctions').checkForMissingTolls;
 // const redisClient = require('../init_redis').redisClient;
-const redisClient = require('../app').redis;
 // const redisClient = null;
 // ENV variables
 const keyGoogle = process.env.GOOGLE_API;
 const keyOpenRoute = process.env.API_OPENROUTES;
+
 /**
  * find a section where there might be a toll
  * return: list of sections
  */
 function findSection (steps, array) {
   const sections = [];
+  const path = [];
+  let start;
+  let end;
+
   for (const i in steps) {
+    start = steps[i].start_location;
+    end = steps[i].end_location;
     // checking for the missing tolls
     for (const idx in array) {
-      if ((steps[i].start_location.lat === array[idx].surNorte.lat && steps[i].start_location.lng === array[idx].surNorte.lng) || (steps[i].start_location.lat === array[idx].norteSur.lat && steps[i].start_location.lng === array[idx].norteSur.lng)) {
+      if ((steps[i].start_location.lat === array[idx].surNorte.lat &&
+          steps[i].start_location.lng === array[idx].surNorte.lng) ||
+          (steps[i].start_location.lat === array[idx].norteSur.lat &&
+          steps[i].start_location.lng === array[idx].norteSur.lng)) {
         array[idx].inRoute = true;
       }
     }
     if (steps[i].html_instructions.includes('Toll')) {
       /* iterate throught the result of the road and return it */
       sections.push({
         distance: steps[i].distance.text,
-        start_location: steps[i].start_location,
-        end_location: steps[i].end_location
+        start_location: start,
+        end_location: end
       });
     }
+    path.push({ start, end });
   }
-  return sections;
+
+  return {
+    tolls: sections,
+    path: path
+  };
 }
```

Comment:
# last change
It returns just two variables, the section and the path for not make another circle